pixel: Remove pixel_experience files older 2016

sergoops opened the pull request:
This allows to upload ucompressed photos/videos in Google Photos

sergoops committed:
pixel: Remove pixel_experience files older 2016
This allows to upload ucompressed photos/videos in Google Photos

sergoops commented:
With this files (pixel_experience > 2016) it's possible to upload only compressed photos and videos

gotenksIN commented:
we don't seem to have this problem, on mi11x uncompressed images were being backed up just fine

are you sure its not because of the 2016 one?

sergoops commented:
> we don't seem to have this problem, on mi11x uncompressed images were being backed up just fine
> 
> are you sure its not because of the 2016 one?

This problem there is in my previous builds (from february) and in today builds. I read information about pixels from google website and found that only Pixel 1 have not limits to upload uncompressed photos/videos. It's was released in 2016, so i deleted pixel feature xmls for pixels released in 2017+ and problem has gone.

Previously i used only "pixel_2016_exclusive.xml" to get unlimited and uncompressed gphoto. It seems that files which i deleted in this commit overwrites some logic in gphoto and it (gphoto) incorrectly defines Pixel model.

My device is oneplus 9rt, but i don't think it's have matter.

sergoops commented:
I am not sure, that this files doesn't enable some useful features. But gphoto now working correctly.
